src/model.rs

```diff
@@ -385,13 +398,16 @@ mod tests {
         let state = model.forward(delta_ts, ratings, None);
         let stability = state.stability.to_data();
         let difficulty = state.difficulty.to_data();
-        assert_approx_eq(
-            stability.to_vec::<f32>().unwrap().try_into().unwrap(),
-            [0.2619, 1.7074, 5.8691, 25.0124, 0.2859, 2.1482],
+        assert_abs_diff_eq!(
+            stability.to_vec::<f32>().unwrap().as_slice(),
+            [
+                0.23664382, 1.9285083, 6.27545, 26.054081, 0.23664382, 2.3679762
+            ]
+            .as_slice(),
```

Comment:
```suggestion
            &[
                0.23664382, 1.9285083, 6.27545, 26.054081, 0.23664382, 2.3679762
            ]
```

can this work?

Reply:
<img width="716" alt="image" src="https://github.com/user-attachments/assets/bdb63dfa-d66a-495c-abf4-e79a3a3ba1b8" />

It doesn't.

Reply:
```suggestion
            [
                0.23664382, 1.9285083, 6.27545, 26.054081, 0.23664382, 2.3679762
            ]
```

then this?

since we can `assert_eq!(vec![1,2], [1,2]);`

Reply:
<img width="601" alt="image" src="https://github.com/user-attachments/assets/7e5fe15e-8906-4f17-953a-250b05fde4c1" />